sdk/search/azure-search/src/main/java/com/azure/search/models/VisualFeature.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for VisualFeature.
+ */
+public enum VisualFeature {
```

Comment:
There are a couple more supported values that will be supported with this as part of the latest Azure Search deployment that is currently rolling out.  Would obviously require we update the swagger with them.  Not a big deal if they don't make it into this release, but just worth mentioning, I had had it on my radar to add them soon and we ended up assigning it as an intro task for Jennifer but I'm not sure what the timeline will be on cutting a new release or not.

Also, I think this should be string constants instead of an enum

Reply:
@caryjmac What are your thoughts on a policy for how to handle updates like these? If you only update the Swagger for the API version where the new skills/types/properties are introduced, then older versions of the SDK will break ungracefully when trying to deserialize them. On the other hand, the best case scenario I can think of is that you implement validation to return 400 on GET if unsupported skills are used with an older API version, but that's still breakage from the client perspective (just probably with a better error message).

Curious what your team decides on this. It's important to figure this out given the renewed focus on maintaining backward compatibility in the new API guidelines.

Reply:
Perhaps we should just change these types to strings then, and have a list of supported options as constants, so that even in SDK versions where a type is "unsupported" it will at least deserialize into a string.  We assume these are backward compatible since all we're doing is adding new options to a list of a preexisting property on the skill.  And we expect this to happen frequently given that cognitive services releases new features and options all the time that we want to be able to consume, so....

Reply:
@careyjmac My question was more general than just the enums, but in that specific case I agree with you.

Let's not block this PR on those Swagger changes though.